mlflow/store/file_store.py

```diff
@@ -31,18 +31,39 @@ def _default_root_dir():
     return get_env(_TRACKING_DIR_ENV_VAR) or os.path.abspath(DEFAULT_LOCAL_FILE_AND_ARTIFACT_PATH)
 
 
+def _make_persisted_experiment_dict(experiment):
+    exp_dict = dict(experiment)
+    try:  # Cast to int for backwards compatibility with type change for run submit
```

Comment:
What breaks if we don't do all this manual munging of dictionary experiments?

Reply:
Remote execution from new version to old version. Somehow, experiment/run_infos are copied into the docker container. But the version in the remote machine does not match